recipes/file/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "file" %}
+{% set version = "5.37" %}
+{% set sha256 = "e9c13967f7dd339a3c241b7710ba093560b9a33013491318e88e6b8b57bae07f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.astron.com/pub/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  run:
```

Comment:
```suggestion
```

Reply:
Thanks! Apparently, it caused failures on the initial build.